PWGJE/EMCALJetTasks/AliAnalysisTaskJetCharge.h

```diff
@@ -1,5 +1,5 @@
-#ifndef ALIANALYSISTASKJETCHARGE_H
-#define ALIANALYSISTASKJETCHARGE_H
+#ifndef AliAnalysisTaskJetCharge_H
+#define AliAnalysisTaskJetCharge_H

```

Comment:
These header guard macros should be all caps by convention. It's not worth holding up this PR, but please change this back next time you prepare a PR